crates/holochain/src/conductor/conductor/tests.rs

```diff
@@ -473,7 +473,11 @@ async fn test_signing_error_during_genesis() {
         panic!("this should have been an error")
     };
 
-    assert_matches!(err, ConductorApiError::ConductorError(ConductorError::GenesisFailed { errors }) if errors.len() == 1);
+    if let ConductorApiError::ConductorError(inner) = err {
+        assert_matches!(*inner, ConductorError::GenesisFailed { errors } if errors.len() == 1);
+    } else {
+        panic!("this should have been an error too");
+    }
```

Comment:
Oh, haha, I just noticed this is exactly what you changed it from. Hmm, why would clippy complain about this?

Reply:
i think its because the `ConductorError` variant changed size and so i boxed it and we can't match on boxes, so this syntax mirrors the error/panic handling immediately above on lines ~470